tensorboard/data_compat.py

```diff
@@ -81,10 +81,27 @@ def make_summary(tag, metadata, data):
 
 
 def _migrate_histogram_value(value):
+    """Convert `old-style` histogram value to `new-style`.
+
+    Since by default min value is DBL_MAX and max value is -DBL_MAX, empty
```

Comment:
I might phrase this a little differently - in particular, "min" and "max" here might be easy to confuse with the legacy histogram format's actual min and max fields (which are derived from the data), but I think you're actually referring to the minimum and maximum bucket limits?  (Also, I think you have -DBL_MAX and DBL_MAX swapped.)

Maybe something like:

> The "old-style" format can have outermost bucket limits of -DBL_MAX and DBL_MAX,
> which are problematic for visualization. We replace those here with the actual min and
> max values seen in the input data, but then in order to avoid introducing "backwards"
> buckets (where left edge > right edge), we first must drop all empty buckets on the
> left and right ends.

Reply:
This is referring to https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/lib/histogram/histogram.cc#L93, when `_min` and `_max` aren't set given empty buckets, I think the limits could end up being [DBL_MAX, -DBL_MAX], but I agree that it's very confusing, adopted the suggested doc string, thanks!

Reply:
Ohhh right I understand now, sorry for the confusion! But yes, since in the all-empty histogram case now, we should be ignoring the legacy format's min and max anyway (since we just produce the [0, 3] empty tensor), I think the fact that those values might be [DBL_MAX, -DBL_MAX] hopefully shouldn't be an issue.